src/components/AddComment.jsx

```diff
@@ -0,0 +1,92 @@
+import { useState } from "react";
+import { postComment } from "../api";
+
+
+
+const AddComment = ({ article_id, setComments, }) => {
+  const [newBody, setNewbody] = useState("")
+    const [error, setErr] = useState(null)
+
+   
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (!newBody)  {
```

Comment:
This is a good validation check, stops the backend recieving a lot of blank comments from misclicks